libs/threading_base/include/hpx/threading_base/print.hpp

```diff
@@ -126,31 +125,48 @@ namespace hpx { namespace debug {
         inline std::ostream& operator<<(
             std::ostream& os, const current_time_print_helper&)
         {
-            static std::chrono::high_resolution_clock::time_point log_t_start =
-                std::chrono::high_resolution_clock::now();
+            using namespace std::chrono;
+            static steady_clock::time_point log_t_start =
+                steady_clock::now();
             //
-            auto now = std::chrono::high_resolution_clock::now();
-            auto nowt = std::chrono::duration_cast<std::chrono::microseconds>(
-                now - log_t_start)
-                            .count();
+            auto now = steady_clock::now();
+            auto nowt = duration_cast<microseconds>(now - log_t_start).count();
             //
             os << debug::dec<10>(nowt) << " ";
             return os;
         }
 
+#ifdef HPX_HAVE_CXX17_FOLD_EXPRESSIONS
         template <typename... Args>
-        void display(const char* prefix, Args... args)
+        void display(const char* prefix, std::forward<Args>(args)... args)
```

Comment:
This doesn't look right.

Reply:
It isn't right. not sure how that crept in, but I've removed it.